pkgs/tools/text/gucci/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  name = "gucci-${version}";
+  version = "0.1.0";
+
+  goPackagePath = "github.com/noqcks/gucci";
+
+  src = fetchFromGitHub {
+    owner = "noqcks";
+    repo = "gucci";
+    rev = version;
+    sha256 = "0ksrmzb3iggc7gm51fl0jbb15d0gmpclslpkq2sl2xjzk29pkllq";
+  };
+
+  goDeps = ./deps.nix;
+
+  buildFlags = "--tags release";
```

Comment:
Is there a way we can enrich this to inject the version of the package into the build? Right now the finished binary returns a wrong version:

```
❯ result-bin/bin/gucci --version
gucci version 0.0.0-dev.0
```

Reply:
Actiually, what does `--tags release` even do? I don't see it in the upstream travis spec.

I think this will work for injecting the version:

```
  buildFlagsArray = ''
    -ldflags=-X main.AppVersion=${version}
  '';
```

Reply:
> Actiually, what does --tags release even do? I don't see it in the upstream travis spec.
That was a left-over from the Go package instructions in the Nix manual. You're correct that it accomplishes nothing here. Removed!

> I think this will work for injecting the version
Added!